recipe/meta.yaml

```diff
@@ -47,10 +47,9 @@ source:
 {% else %}
   - url: https://www.python.org/ftp/python/{{ version }}/Python-{{ version }}{{ dev }}.tar.xz
     # md5 from: https://www.python.org/downloads/release/python-{{ ver3nd }}/
-    md5: ecc29a7688f86e550d29dba2ee66cf80
+    md5: fddb060b483bc01850a3f412eea1d954
 {% endif %}
     patches:
-      - patches/0000-Fix-off-by-one-error-in-_winapi_WaitForMultipleObjec.patch
```

Comment:
@isuruf is the preferred pattern here to delete the patch from the meta.yaml but leave it in the recipe/patches/ directory? (i.e., should I back out my deletion of the associated file?)